integration/src/test/resources/non-blocking-test-config-global-err.xml

```diff
@@ -255,16 +255,72 @@
 
     <flow name="foreach" initialState="stopped">
         <test:assert-intercepting responseSameTask="false"/>
+        <test:non-blocking-processor/>
         <foreach>
-            <test:assert-intercepting responseSameTask="false" count="2"/>
+            <test:assert-intercepting responseSameTask="false"/>
             <test:non-blocking-processor/>
             <foreach>
+                <test:assert-intercepting responseSameTask="false"/>
+                <http:request config-ref="http-request-config" path="/test" method="POST" >
```

Comment:
Create a jira and add a TODO to migrate these http requests to a sdk extension for testing.